homeassistant/components/neato/button.py

```diff
@@ -0,0 +1,55 @@
+"""Support for Neato buttons."""
+from __future__ import annotations
+
+import logging
+
+from pybotvac import Robot
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import NEATO_DOMAIN, NEATO_ROBOTS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Neato button from config entry."""
+    dev = []
+
+    for robot in hass.data[NEATO_ROBOTS]:
+        dev.append(NeatoDismissAlertButton(robot))
+
+    if not dev:
+        return
+
+    _LOGGER.debug("Adding vacuum buttons %s", dev)
+    async_add_entities(dev, True)
```

Comment:
```suggestion
    entities = [
        NeatoDismissAlertButton(robot)
        for robot in hass.data[NEATO_ROBOTS]
    ]

    _LOGGER.debug("Adding vacuum buttons %s", entities)
    async_add_entities(entities, True)
```

Reply:
Not sure that debugging is actually providing something valuable?

Reply:
I removed the debug, I kept it since I thought I is good practice to have it watching around